apps/site/src/app/portal/Portal.tsx

```diff
@@ -1,21 +1,62 @@
-import api from "@/lib/utils/api";
+"use client";
```

Comment:
Issue: I should've said this in my previous comment, but now that you're using `redirect`, you don't need "use client" or `useEffect`. You can directly check if `status` is `null` and redirect based off of that.